### PR TITLE
Task completed

### PR DESCRIPTION
**Pull request by RUnmesh**

### CODE REVIEW

File: Unmesh/README.md [hunk 1]
```diff
@@ -9,5 +9,8 @@
 - Update the below section with your progress
 
 ## Progress
-Example:
-1. [name] ([github nick]) [pset submitted]
+1. Unmesh Roy (RUnmesh) pset1
+2. Unmesh Roy (RUnmesh) pset2
+3. Unmesh Roy (RUnmesh) pset3
+4. Unmesh Roy (RUnmesh) pset4
+5. Unmesh Roy (RUnmesh) pset5
```
Comment (twitu) [inline Unmesh/README.md]: single line for latest progress

**Comment by twitu** [comment]
remove all binaries, .txt, .bmp files and test cases.
use .gitignore to add such files to ignore list

File: Unmesh/pset5/speller/dictionary.c [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+#include<stdio.h>
+#include<string.h>
+
+#include "dictionary.h"
+
+int count=0;
+
+typedef struct node
+{
+   int valid;
```
Comment (twitu) [inline Unmesh/pset5/speller/dictionary.c]: valid variable can be of type boolean.

Comment (RUnmesh) [reply Unmesh/pset5/speller/dictionary.c]: Thanks.I'll make the changes.

**Comment by siddhantkhandelwal** [comment]
@RUnmesh There are merge conflicts in the Pull Request.
Refer: https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts

**Comment by siddhantkhandelwal** [comment]
Updates @RUnmesh ?

**Pull request closed without merging**